### PR TITLE
mp-b2c-refactoring

### PR DESCRIPTION
**Pull request by himanshu-saxena-metacube**
mp-b2c-refactoring - https://spryker.atlassian.net/browse/CC-16763

### CODE REVIEW

**Comment by himanshu-saxena-metacube** [comment]
[results.zip](https://github.com/spryker/robotframework-suite-tests/files/8939429/results.zip)

File: resources/environments/environments_api_mp_b2c.json [hunk 1]
```diff
@@ -1,306 +1,382 @@
...
+            "sku": "016",
+            "name": "Sony Cyber-shot DSC-W800"
+        },
+        "alternative_abstract_product": {
+            "price_chf": "39675",
+            "sku": "134"
+        },
+        "abstract_product": {
+            "product_with_multiple_variants": {
+                "sku": "130"
```
Comment (OKrivtsova) [inline resources/environments/environments_api_mp_b2c.json:353]: we have other variables about multivariants. please consolidate them:
 "multivariant": {
                "variant1_name": "Lenovo ThinkStation P300",
                "variant1_sku": "130_29285281",
                "variant2_sku": "130_24326086"
            },
and

       "special_product_abstract_sku": {
            "with_multiple_variants": "130",
            "with_measurement_unit": "M23723",
            "with_packaging_unit": "M21766"

File: resources/common/common_api.robot [hunk 2]
```diff
@@ -1509,21 +1509,4 @@ Cleanup all availability notifications:
                 ${availability_notification_id}=    Replace String    ${availability_notification_id}    ]   ${EMPTY}
                 Log    availability_notification_id: ${availability_notification_id}
                 ${response_delete}=    DELETE    ${current_url}/availability-notifications/${availability_notification_id}    headers=${headers}    timeout=${api_timeout}    allow_redirects=${default_allow_redirects}    auth=${default_auth}    expected_status=204
-        END
-
-Update order status in Database:
```
[the remark this answers is not in the document]
Comment (himanshu-saxena-metacube) [reply resources/common/common_api.robot:1514]: Fixed

File: resources/environments/environments_api_mp_b2c.json (hunk 1 above)
Comment (himanshu-saxena-metacube) [reply resources/environments/environments_api_mp_b2c.json]: Fixed

**Review by OKrivtsova — APPROVED** [review]

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**